README.md

```diff
@@ -0,0 +1,48 @@
+# PhpToken Polyfill
+
+A polyfill to bring the [PHP 7.3 `array_key_first`](https://www.php.net/manual/en/function.array-key-first.php) and [PHP 7.3 `array_key_last`](https://www.php.net/manual/en/function.array-key-last.php) functions to PHP 5.3 and later.
+
+[![Latest Stable Version](https://poser.pugx.org/phpwatch/array_key_first_last-polyfill/v/stable)](https://packagist.org/packages/phpwatch/array_key_first_last-polyfill) [![License](https://poser.pugx.org/phpwatch/array_key_first_last-polyfill/license)](https://github.com/phpwatch/array_key_first_last-polyfill)  [![CI](https://github.com/phpwatch/array_key_first_last-polyfill/workflows/CI/badge.svg)](https://github.com/phpwatch/array_key_first_last-polyfill/actions)
+
```

Comment:
After #6, perhaps we can add a build status badge here too?

Reply:
Yes. After #6 has been resolved, these badges will be worked.